tensorflow/tools/ci_build/builds/pip.sh

```diff
@@ -131,15 +131,19 @@ echo "Using Bazel flags: ${BAZEL_FLAGS}"
 
 PIP_BUILD_TARGET="//tensorflow/tools/pip_package:build_pip_package"
```

Comment:
@av8ramit are we using this script?

for rocm, we may want to remove this script from the critical path.